scripts/validate_documentation.py

```diff
@@ -0,0 +1,227 @@
+import argparse
+from fnmatch import fnmatch
+import json
+import os
+import pickle
+import re
+import sys
+
+import docutils.nodes
```

Comment:
docutils is part of the standard library, right? why the blank line if that's the case?